pkg/proof/proof.go

```diff
@@ -24,7 +23,7 @@ import (
 
 // NewTxInclusionProof returns a new share inclusion proof for the given
 // transaction index.
-func NewTxInclusionProof(codec rsmt2d.Codec, data types.Data, txIndex uint64) (types.ShareProof, error) {
+func NewTxInclusionProof(data types.Data, txIndex uint64) (types.ShareProof, error) {
```

Comment:
thanks for removing this since we weren't using this!

out of curiosity, was there any reason in node to remove this as well?

Reply:
To be honest, I have no idea even when node removed this 😓